modules/julia/CMakeLists.txt

```diff
@@ -0,0 +1,142 @@
+
+if(NOT Julia_FOUND)
```

Comment:
`HAVE_JULIA` due to OpenCV variables convention (see below)

Reply:
`Julia_FOUND` comes from `find_package`. I'm not sure how to change that. I'm changing `Julia_BUILD` to `HAVE_JULIA`. This is also how it's done in the matlab bindings.

Reply:
Please use `Julia_FOUND` in "FindJulia.cmake" only.
This file should setup `HAVE_JULIA` variable if dependency has found and it passes basic checks which are not covered by `find_package()` (like Julia version or availability required Julia modules - these checks may be adjusted later, so we don't use `FOUND` variables directly).

Outside of this file (auto-detection script) use `HAVE_JULIA`.

Reply:
**UPD**: Fixed file name

Reply:
Done